tests/test_core.py

```diff
@@ -632,3 +627,25 @@ def test_save_subtitles_single_directory_encoding(movies, tmpdir):
     path = os.path.join(str(tmpdir), os.path.splitext(os.path.split(movies['man_of_steel'].name)[1])[0] + '.srt')
     assert os.path.exists(path)
     assert io.open(path, encoding='utf-8').read() == u'ハローワールド'
+
+
+def test_scan_archive_with_no_video():
+    rar = os.path.realpath(os.path.join('tests', 'data', 'rar', 'README.rar'))
```

Comment:
What's inside of the real rar files? Are they big? Is there a way to test without actually having them?